doc/sphinx/proof-engine/ltac.rst

```diff
@@ -532,42 +635,47 @@ tactic that succeeds from a list of tactics:
 
             Tactic Notation "foo" tactic_list(tacs) := first tacs.
 
-Left-biased branching: ||
-~~~~~~~~~~~~~~~~~~~~~~~~~
+First tactic to make progress: ||
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Yet another way of branching without backtracking is the following
 structure:
 
-.. tacn:: @ltac_expr__1 || @ltac_expr__2
-   :name: || (left-biased branching)
+.. tacn:: @ltac_expr1 || {| @ltac_expr2 | @binder_tactic }
```

Comment:
Again, why the lack of subscripts?

Reply:
Same thing, that from the grammar production.